include/lsst/base.h

```diff
@@ -29,23 +29,6 @@
  */
 #include <memory>
 
-/**
- * A shared pointer to an object
- *
- * \note Using this macro is preferable to the Ptr typedef in type T as no definition of T need be provided,
- * a forward definition (<tt>class T;</tt>) is sufficient
- *
- * \sa CONST_PTR
- */
-#define LSST_WHITESPACE /* White space to avoid swig converting vector<PTR(XX)> into vector<shared_ptr<XX>> */
-#define PTR(...) std::shared_ptr<__VA_ARGS__ LSST_WHITESPACE > LSST_WHITESPACE
-/**
- * A shared pointer to a const object
- *
- * \sa PTR
- */
-#define CONST_PTR(...) std::shared_ptr<const __VA_ARGS__ LSST_WHITESPACE > LSST_WHITESPACE
-
```

Comment:
Please also remove the references from the [Doxygen configuration](https://github.com/lsst/base/blob/tickets/DM-6056/doc/base.inc#L2169-L2170). A stack-wide text search tool may be handy, if you have one.